src/activity_handlers.cpp

```diff
@@ -2626,8 +2624,12 @@ void activity_handlers::try_sleep_finish( player_activity *act, player *p )
 
 void activity_handlers::craft_do_turn( player_activity *act, player *p )
 {
-    const recipe &rec = recipe_id( act->name ).obj();
+    const int pos = act->values.front();
+    item &craft = p->i_at( pos );
```

Comment:
There's a potential bug here: you can no longer have the item at this point (for example, a zed ripped off your backpack and the item has dropped).

Reply:
Noted, I'll add a check.